docs/src/pages/components/data-grid/selection/ControlledSelectionGrid.tsx

```diff
@@ -10,15 +10,16 @@ export default function ControlledSelectionGrid() {
   });
 
   // TODO: https://github.com/mui-org/material-ui-x/issues/246
```

Comment:
It looks like you have the answer to your question :p

```suggestion
```

<img width="342" alt="Capture d’écran 2021-02-06 à 17 33 16" src="https://user-images.githubusercontent.com/3165635/107124023-6b0d7700-68a1-11eb-9188-3038151d0033.png">